community/cypher/interpreted-runtime/src/test/scala/org/neo4j/cypher/internal/runtime/interpreted/TransactionBoundPlanContextTest.scala

```diff
@@ -111,4 +112,106 @@ class TransactionBoundPlanContextTest extends CypherFunSuite {
     transactionalContext.close(true)
     transaction.close()
   }
+
+  test("indexesGetForLabel should return both regular and unique indexes") {
+    val graph = new GraphDatabaseCypherService(database)
+
+    val setupTransaction = graph.beginTransaction(explicit, AUTH_DISABLED)
+    database.schema().indexFor(Label.label("L1")).on("prop").create()
+    database.schema().indexFor(Label.label("L2")).on("prop").create()
+    database.schema().constraintFor(Label.label("L1")).assertPropertyIsUnique("prop2").create()
+    database.schema().constraintFor(Label.label("L2")).assertPropertyIsUnique("prop2").create()
+    setupTransaction.success()
+    setupTransaction.close()
+
```

Comment:
Here we need to wait for the non-unique indexes to come online, otherwise this test could become flaky. Same for other tests in this class.

Otherwise, nice tests!